### PR TITLE
Az

### PR DESCRIPTION
**Pull request by adam50200**
the code is written in Azerbaijani

### CODE REVIEW

**Comment by adam50200** [comment]
who is the admin or mod here?

**Comment by adam50200** [comment]
Did I do it right?

**Comment by adam50200** [comment]
What next should I do?

**Comment by getneil** [comment]
@adam50200 include also the index.ts file update.  please follow the instruction in https://github.com/pkgxdev/ossapp#readme

thanks.

**Comment by getneil** [comment]
> > @adam50200 include also the index.ts file update. please follow the instruction in https://github.com/pkgxdev/ossapp#readme
> > thanks.
> 
> How? I don’t understand

in the link i provided here is the screenshot of the instruction
Screenshot 2023-12-05 at 8 45 40 AM

File: svelte/src/libs/translations/languages/translation to Azerbaijan [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+"Populyarlıq" :" ən populyar",
+"ən son": "ən son"
+    },
+"ümumi": {
+      "home": "Ana səhifə",
+"all":"All",
+"articles":"Məqalələr",
+"seminar": "seminarlar",
+"metadata": "metadata",
+"Ana səhifə": "Ana səhifə",
```
Comment (jhheider) [inline svelte/src/libs/translations/languages/translation to Azerbaijan:51]: Only the mapping (right side) should be changed. The key (left side) must remain in English to be looked up.

File: svelte/src/libs/translations/languages/translation to Azerbaijan [hunk 2]
```diff
@@ -0,0 +1,101 @@
+{
+  "id": "en",
+"label": "English",
```
Comment (jhheider) [inline svelte/src/libs/translations/languages/translation to Azerbaijan:3]: These should be changed to the appropriate language

**Review by jhheider — COMMENTED** [review]
The file should also be named `az.json`.

File: svelte/src/libs/translations/languages/translation to Azerbaijan (hunk 2 above)
Comment (adam50200) [reply svelte/src/libs/translations/languages/translation to Azerbaijan:3]: Let's leave it, I couldn't, I couldn't do it. is there any other activity easier?

Comment (adam50200) [reply svelte/src/libs/translations/languages/translation to Azerbaijan:3]: Just give me contributor role😐
For my efforts

**Comment by mxcl** [comment]
Reopen if you

**Comment by mxcl** [comment]
k closed as I thought it didn't include the translation, but we should get this over the line since that work was done.

**Comment by mxcl** [comment]
Closing as the original author is now hostile to the project.

**Pull request closed without merging**